centreon/www/include/common/common-Func.php

```diff
@@ -1552,21 +1552,31 @@ function get_child($id_page, $lcaTStr)
     global $pearDB;
 
     if ($lcaTStr != "") {
-        $rq = " SELECT topology_parent,topology_name,topology_id,topology_url,topology_page,topology_url_opt, is_react
-                FROM topology
-                WHERE topology_page IN ($lcaTStr)
-                AND topology_parent = '" . $id_page . "' AND topology_page IS NOT NULL AND topology_show = '1'
-                ORDER BY topology_order, topology_group ";
+        $rq = <<<SQL
+            SELECT topology_parent, topology_name, topology_id, topology_url, topology_url_substitute,
+                topology_page, topology_url_opt, is_react
+            FROM topology
+            WHERE topology_page IN ($lcaTStr)
+                AND topology_parent = :id_page
+                AND topology_page IS NOT NULL
+                AND topology_show = '1'
+            ORDER BY topology_order, topology_group
```

Comment:
Is this OK with mysql8 ?

Reply:
Yes, otherwise the legacy pages would not be displayed at all